src/lib/public/board/data/officers.json

```diff
@@ -1,718 +1,368 @@
-    "picture": "aaron-lieberman.webp"
+      "S21": { "title": "Internal Vice President, nodebuds Officer", "tier": 1 },
+      "F21": { "title": "President, nodebuds Officer", "tier": 0 },
+      "S22": { "title": "President, nodebuds Officer", "tier": 0 },
+      "F22": { "title": "Special Events Team Lead", "tier": 6 }
+    }
   },
   {
     "fullName": "Shaleen Mathur",
-    "positions": {
-      "S21": {
-        "title": "Workshop Manager",
-        "tier": 6
-      }
-    },
-    "picture": "shaleen-mathur.webp"
+    "picture": "shaleen-mathur.webp",
+    "positions": { "S21": { "title": "Workshop Manager", "tier": 18 } }
   },
   {
     "fullName": "Ean McGilvery",
-    "positions": {
-      "S21": {
-        "title": "nodebuds Officer",
-        "tier": 23
-      }
-    },
-    "picture": "ean-mcgilvery.webp"
+    "picture": "ean-mcgilvery.webp",
+    "positions": { "S21": { "title": "nodebuds Officer", "tier": 22 } }
   },
   {
     "fullName": "Jorge Mejia",
+    "picture": "jorge-mejia.webp",
     "positions": {
-      "F21": {
-        "title": "Dev Officer",
-        "tier": 21
-      },
-      "S22": {
-        "title": "Dev Officer",
-        "tier": 21
-      },
-      "F22": {
-        "title": "Design Officer",
-        "tier": 17
-      }
-    },
-    "picture": "jorge-mejia.webp"
+      "F21": { "title": "Dev Officer", "tier": 19 },
+      "S22": { "title": "Dev Officer", "tier": 19 },
+      "F22": { "title": "Design Officer", "tier": 16 }
+    }
   },
   {
     "fullName": "Serena Naranjo",
+    "picture": "serena-naranjo.webp",
     "positions": {
-      "F21": {
-        "title": "Create Officer",
-        "tier": 17
-      },
-      "S22": {
-        "title": "Create Officer",
-        "tier": 17
-      }
-    },
-    "picture": "serena-naranjo.webp"
+      "F21": { "title": "Create Officer", "tier": 16 },
+      "S22": { "title": "Create Officer", "tier": 16 }
+    }
   },
   {
     "fullName": "Dalisa Nguyen",
-    "positions": {
-      "S21": {
-        "title": "nodebuds Officer",
-        "tier": 23
-      }
-    },
-    "picture": "dalisa-nguyen.webp"
+    "picture": "dalisa-nguyen.webp",
+    "positions": { "S21": { "title": "nodebuds Officer", "tier": 22 } }
   },
   {
     "fullName": "Kayla Nguyen",
-    "positions": {
-      "F21": {
-        "title": "Create Officer",
-        "tier": 17
-      }
-    },
-    "picture": "kayla-nguyen.webp"
+    "picture": "kayla-nguyen.webp",
+    "positions": { "F21": { "title": "Create Officer", "tier": 16 } }
   },
   {
     "fullName": "Jacob Nguyen",
+    "picture": "jacob-nguyen.webp",
     "positions": {
-      "S21": {
-        "title": "President, Create Director, nodebuds Officer",
-        "tier": 0
-      },
-      "F22": {
-        "title": "Treasurer",
-        "tier": 4
-      }
-    },
-    "picture": "jacob-nguyen.webp"
+      "S21": { "title": "President, Create Director, nodebuds Officer", "tier": 0 },
+      "F22": { "title": "Treasurer", "tier": 4 }
```

Comment:
The updated "tier" for Treasurer is '3' 

`"Treasurer": { "id": 3, "index": 250 }`